src/hotspot/share/classfile/stackMapFrame.cpp

```diff
@@ -188,6 +201,14 @@ bool StackMapFrame::is_assignable_to(
     return false;
   }
 
+  // Check that assert unset fields are compatible
+  bool compatible = is_unset_fields_compatible(target->assert_unset_fields());
```

Comment:
We decided 'is' was more consistent but it does read funny.  Maybe verify_unset_fields_are_compatible ?

Reply:
Maybe `verify_unset_fields_compatibility` works?